src/dependencies.jl

```diff
@@ -75,26 +75,16 @@ function clone_all_registries(f::Function, registry_list::Vector{Pkg.RegistrySpe
     end
 end
 
-function get_latest_version!(deps, registries; options::Options)
-    for registry in registries
-        registry_toml_path = joinpath(registry, "Registry.toml")
-        registry_toml = TOML.parsefile(registry_toml_path)
-        packages = registry_toml["packages"]
+function get_latest_version!(deps::Set{DepInfo}, registries::Vector{RegistryInstances.RegistryInstance}; options::Options)
+    for registry_instance in registries
+        packages = registry_instance.pkgs
 
         for dep in deps
-            uuid = string(dep.package.uuid)
+            uuid = dep.package.uuid
```

Comment:
Can you add a type assertion here, to make sure that it's what we expect?

Reply:
I don't think that field can be anything but an `UUID` https://github.com/JuliaRegistries/CompatHelper.jl/blob/cb50eedfe076b3045fc3a1da424011035435fa0e/src/utilities/types.jl#L10